src/rules/noRequireImportsRule.ts

```diff
@@ -26,27 +26,20 @@ export class Rule extends Lint.Rules.AbstractRule {
 }
 
 class NoRequireImportsWalker extends Lint.RuleWalker {
-    public visitVariableStatement(node: ts.VariableStatement) {
-        const declarations = node.declarationList.declarations;
-        for (let decl of declarations) {
-            this.handleDeclaration(decl);
+    public visitCallExpression(node: ts.CallExpression) {
```

Comment:
yeah, I would consider that bad form if you're not in a node environment. and it's simpler for TSLint not to ask about your project's environment (browser vs. node)